spec/message_configuration_client_spec.rb

```diff
@@ -193,4 +196,18 @@
       expect(subject.send(:fetch_truststore_from_server)).to be_nil
     end
   end
+
+  describe "#restart_evmserverd" do
```

Comment:
Should these examples have an expectation on `restart` being called?

Reply:
@bdunne Agreed but I think the do. Shouldn't this do it:

209       expect(LinuxAdmin::Service).to receive(:new).with("evmserverd").and_return(double(@spec_name, :running? => false))

Reply:
That just means that calling `restart` on that `double` will return `nil`, right?

Reply:
I don't think it set up any expectation that `restart` gets called

Reply:
I get it now. Thank you! Will fix.